peewee_migrate/cli.py

```diff
@@ -85,15 +86,30 @@ def create(name, database=None, auto=False, auto_source=False, directory=None, v
 
 
 @cli.command()
-@click.argument('name')
+@click.argument('name', required=False, default='1')
```

Comment:
May be add something like this?

```python
@click.option('--count', default=1, help='Number of migrations to rollback. Starting from last one')
```

And make `name` argument optional like you suggest. I think it will be more clear

Reply:
I thought about that, but not sure about the best way. The way when we can pass it to name argument is just better in terms of less typing. Sqlalchemy does the similar, but they ask you to type something like `rollback -4`. However click does not allow us to do so, as far as I know.

Reply:
Yea, typing `--count` may be worst. I thinking about short key like `-c`

Then default behaviour will be same: pw_migrate rollaback == pw_migrate rollback -c 1

But click does not support mutual exclusive groups and don't help write correct interface (disallow `name` when `count` setting up).

So, let's make it like you suggest :)

----

What about `-4` - it's possible, but in unexpected way:

`pw_migrate rollback -- -4`
https://click.palletsprojects.com/en/7.x/arguments/#option-like-arguments

Reply:
Only one thing to change is regex.

Better and simple is use str.isdigit() for that case, i guess

Reply:
It looks more strange:) 
Maybe we leave it as is for now. So default is the same as `rollback --count 1`?  The regex in current version is not needed, because if we specify it as separate option, we may use click's type, so it converts option from stdin to integer itself.

Most often what is handy during working on new migration is to be able to simply do `migrate/rollback` without args to check the flow.

Reply:
Ok, but add help text to `count` option about ignoring this option when `name` argument specified. I'm worried about this usage:

`pw_migrate rollback --count 2 very_last_migration`